### PR TITLE
S109_ Load container based on env

### PR DESCRIPTION
**Pull request by pvaass**

### CODE REVIEW

File: Lib/Container/ContainerBuilder.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+namespace CakePHPUtil\Lib\Container;
+
+class ContainerBuilder {
+
+	protected static $_container;
```
Comment (expand-dev) [inline Lib/Container/ContainerBuilder.php]: MINOR Rename this field "$_container" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)

**Comment by expand-dev** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ContainerBuilderComponent.php](https://github.com/ExpandOnline/CakePHPUtil/blob/d48438c9f7806defb6e220a0bb30a4e5e4d640a2/Controller/Component/ContainerBuilderComponent.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MINOR][MINOR] [ContainerBuilderComponent.php](https://github.com/ExpandOnline/CakePHPUtil/blob/d48438c9f7806defb6e220a0bb30a4e5e4d640a2/Controller/Component/ContainerBuilderComponent.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [ContainerBuilderComponent.php#L9](https://github.com/ExpandOnline/CakePHPUtil/blob/d48438c9f7806defb6e220a0bb30a4e5e4d640a2/Controller/Component/ContainerBuilderComponent.php#L9): Rename this field "$_container" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)
1. ![MINOR][MINOR] [ContainerBuilder.php](https://github.com/ExpandOnline/CakePHPUtil/blob/d48438c9f7806defb6e220a0bb30a4e5e4d640a2/Lib/Container/ContainerBuilder.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: Lib/Container/ContainerBuilder.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	protected function _createContainer() {
+		$container = new \Symfony\Component\DependencyInjection\ContainerBuilder();
+		$loader = new \Symfony\Component\DependencyInjection\Loader\YamlFileLoader(
+			$container,
+			new \Symfony\Component\Config\FileLocator(CONTAINER_CONFIG_PATH)
+		);
+
+
+		if(!($env = getenv('SYMFONY_ENV'))) {
+			$env = 'prod';
```
Comment (ronoudgenoeg) [inline Lib/Container/ContainerBuilder.php]: Exception gooien

Comment (pvaass) [reply Lib/Container/ContainerBuilder.php]: @ronoudgenoeg done

Comment (ronoudgenoeg) [reply Lib/Container/ContainerBuilder.php]: @pvaass Vergeten te pushen?

**Comment by expand-dev** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 5 minor

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ContainerBuilderComponent.php](https://github.com/ExpandOnline/CakePHPUtil/blob/8f88b5373d5d662c6c308558b06d2ee20122733b/Controller/Component/ContainerBuilderComponent.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MINOR][MINOR] [ContainerBuilderComponent.php](https://github.com/ExpandOnline/CakePHPUtil/blob/8f88b5373d5d662c6c308558b06d2ee20122733b/Controller/Component/ContainerBuilderComponent.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [ContainerBuilderComponent.php#L9](https://github.com/ExpandOnline/CakePHPUtil/blob/8f88b5373d5d662c6c308558b06d2ee20122733b/Controller/Component/ContainerBuilderComponent.php#L9): Rename this field "$_container" to match the regular expression ^[a-z][a-zA-Z0-9]*$. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS116)
1. ![MINOR][MINOR] [ApiScopeValidator.php](https://github.com/ExpandOnline/CakePHPUtil/blob/8f88b5373d5d662c6c308558b06d2ee20122733b/Lib/Api/Scopes/ApiScopeValidator.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [ContainerBuilder.php](https://github.com/ExpandOnline/CakePHPUtil/blob/8f88b5373d5d662c6c308558b06d2ee20122733b/Lib/Container/ContainerBuilder.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**